record.py

```diff
@@ -31,54 +40,47 @@ def __init__(self):
         self.create_main_panel()
 
         # Timer
-        self.timer = wx.Timer(self)
-        self.Bind(wx.EVT_TIMER, self.on_timer, self.timer)
-        self.rate = SAMPLE_RATE
+        self.rate = IDLE_SAMPLE_RATE
+        self.sample_rate = SAMPLE_RATE
         self.idle_rate = IDLE_SAMPLE_RATE
-        self.timer.Start(self.idle_rate)
-
         self.recording = False
         self.t = 0
+        self.pause_timer = False
+        self.on_timer()
 
+        self.root.mainloop()
 
     def create_main_panel(self):
         # Panels
-        self.img_panel = wx.Panel(self)
-        self.joy_panel = wx.Panel(self)
-        self.record_panel = wx.Panel(self)
-
+        top_half = tk.Frame(self.root)
+        top_half.pack(side=tk.TOP, expand=True, padx=5, pady=5)
+        message = tk.Label(self.root, text="(Note: UI updates are disabled while recording)")
+        message.pack(side=tk.TOP, padx=5)
+        bottom_half = tk.Frame(self.root)
+        bottom_half.pack(side=tk.LEFT, padx=5, pady=10)
+        
         # Images
-        img = wx.Image(320,240)
-        self.image_widget = wx.StaticBitmap(self.img_panel, wx.ID_ANY, wx.Bitmap(img))
-
+        self.img_panel = tk.Label(top_half, image=ImageTk.PhotoImage("RGB", size=IMAGE_SIZE)) # Placeholder
+        self.img_panel.pack(side = tk.LEFT, expand=False, padx=5)
+        
```

Comment:
extra whitespace?

Reply:
Oops, good catch. I think I have VS Code ignoring whitespace in diffs. I'll clean it up soon.